ngrinder-frontend/src/js/main.js

```diff
@@ -66,6 +67,8 @@ axiosInstance.interceptors.request.use(config => {
     return config;
 });
 
+window.jsonlint = jsonlint;
```

Comment:
I think that adding a comment that `window.jsonlint` is used by codemirror json lint addon would make easier to understand why does it need to be declared.